src/main/content/_assets/js/instance-common.js

```diff
@@ -498,3 +508,41 @@ function createSVG(id, classNames, width, height, fill){
         <use xlink:href="/img/carbon-icons/carbon-icons.svg#icon--${id}"></use>
     </svg>`;
 }
+
+function setInstanceModalData(instanceJSON) {
+    if (typeof instanceJSON === "undefined") {
+        console.log("instance data is undefined, cannot load instance");
+        return;
+    }
+
+    let instanceName = instanceJSON.metadata.name;
+    let codereadyVersion = instanceJSON.spec.codeReadyWorkspaces ? instanceJSON.spec.codeReadyWorkspaces.operator.customResourceInstance.devFileRegistryImage.version : null;
+
+    codereadyVersion ? $("#codeready-modal-version").append(`version: <span id='codeready-version-num'>${codereadyVersion}</span>`) : "";
```

Comment:
since there isn't any variable being set, this might make more sense in just a single `if` statement